frontend/src/global_styles/layout/work_packages/_table.sass

```diff
@@ -88,12 +88,12 @@
 .work-packages-split-view--tabletimeline-footer
   // Only add padding to lower/right corner
   // to override default margins
-  .pagination
+  .op-pagination
     margin: 0
     padding: 15px 10px 10px 0
```

Comment:
Margins are perfect here, but paddings should probably not be here, since they're on the inside of the block. I'd put them on `op-pagination` directly. However, The way I'm interpreting these values, I think setting the margin to the value of padding and then padding to 0 would be enough.

Reply:
I hope I didn't miss something, but the styles weren't needed after all. So I removed them completely.